brave/brave-core
# Allow cosmetic filtering to also be applied to 1p items

Resolves https://github.com/brave/brave-browser/issues/8475

## Submitter Checklist:

- Verified that these changes build without errors on
- Verified that these changes pass automated tests (unit, browser, security tests) on

## Test Plan:

- Open brave://settings/shields and confirm that the first dropdown, labeled 'Block trackers & ads', is set to 'Trackers & ads blocked (standard)'.
- For each of the three possible values of the dropdown, visit http://raymondhill.net/ublock/tiles1.html and confirm the two colored tiles at the top of the page display their corresponding colors from the table below. Note that the page will need to be refreshed after each change of the setting. After observing each pair of colors, verify that the first dropdown in the 'Advanced View' of the Shields panel remains consistent with the selected setting.

| Block trackers & ads setting    | Left tile | Right tile |
|---------------------------------------|----------|-----------|
| Trackers & ads blocked (standard) | Green   | May display green for a moment while the page loads, but should remain red afterwards |
| Trackers & ads blocked (aggressive) | Green | Green |
| Allow all trackers & ads  | Red | Red |

- Confirm the behavior in the table above once again, this time changing the setting using the dropdown in the 'Advanced View' of the Shields panel. The page should refresh automatically when the setting is modified. Confirm that the setting in brave://settings/shields is *not* updated by changes to the Shields panel.

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on.
